test/parallel/test-crypto-keygen.js

```diff
@@ -1172,7 +1172,7 @@ function addNumericalSeparator(val) {
       code: 'ERR_OUT_OF_RANGE',
       message:
         'The value of "options.modulusLength" is out of range. ' +
-        'It must be >= 0 && < 4294967296. ' +
+        'It must be >= 0 && <= 4294967295. ' +
         `Received ${addNumericalSeparator(modulusLength)}`
```

Comment:
Can we use this opportunity to remove the message from the test here? Theoretically, any change to existing tests should be treated as semver-major, but it's been decided for a long time that changes to `err.message` should not be considered breaking.
```suggestion
```

Reply:
That's cool, I also felt very strange when I saw ‘err.message’.